SampleCode/QuoteGenerationSample/App/App.cpp

```diff
@@ -44,19 +44,25 @@
 #endif
 
 #include "sgx_urts.h"
+#include "sgx_quote3_defs.h"
```

Comment:
Why not just include [sgx_quote_3.h](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/quote_wrapper/common/inc/sgx_quote_3.h)?

Reply:
The structures defined in sgx_quote_3.h don't define any packing pragmas/attributes. If you try to cast a memory region with a pointer to one of those structs, you get data alignment errors.

Reply:
The data structures in sgx_quote_3.h do have the pack pragmas.  Can you comment more on the alignment errors?

Reply:
Must have been an error on my part. Originally, I copied sgx_quote_3.h (since the include dir is outside the sample code tree) and trimmed it down while testing. I must not have realized I'd deleted the pack pragmas.